### PR TITLE
VB-906: View by visit date small changes

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/views/pages/visits/summary.njk [hunk 1]
```diff
@@ -90,7 +90,7 @@
     <div class="govuk-grid-column-three-quarters">
       <h2 class="govuk-heading-m"><span data-test="visit-room">{% if visitType == 'OPEN' %}Main{% else %}Closed{% endif %}</span> visits room, <span data-test="visit-time">{{ slotTime }}</span></h2>
       <p><span data-test="visit-tables-booked">{{ numberOfResults }} of {{ maxSlots }}</span> tables booked</p>
-      <p><span data-test="visit-visitors-total">{{ totals.visitors }}</span> visitors in total (<span data-test="visit-adults">{{ totals.adults }}</span> adult{% if totals.adults != 1 %}s{% endif %} and <span data-test="visit-children">{{ totals.children }}</span> child{% if (totals.children != 1) %}ren{% endif %})</p>
+      <p><span data-test="visit-visitors-total">{{ totals.visitors }}</span> visitor{% if totals.visitors != 1 %}s{% endif %} (<span data-test="visit-adults">{{ totals.adults }}</span> adult{% if totals.adults != 1 %}s{% endif %} and <span data-test="visit-children">{{ totals.children }}</span> child{% if (totals.children != 1) %}ren{% endif %})</p>
```
Comment (psoleckimoj) [inline server/views/pages/visits/summary.njk:93]: Also made the pluralisation (is that a word?) correct.

**Review by tpmcgowan — COMMENTED** [review]
LGTM

**Comment by psoleckimoj** [comment]
@tpmcgowan for some reason it hasn't accepted it as an approval?

**Review by tpmcgowan — APPROVED** [review]
Forgot to select 'Approve' last time so it just left a comment 🤦‍♂️!

**Pull request merged**